src/components/AddItem.jsx

```diff
@@ -14,6 +15,18 @@ export function AddItem({ listPath }) {
 		);
 
 		try {
+			if (isInputEmpty(itemName)) {
+				formRef.current.reset();
+				return alert('Cannot submit without an item name');
```

Comment:
I think validating on the `submit` event is totally valid, but I think we will want to replace the `alert` with something more customizable down the road.

Also, in this instance (where the validation is specifically about an input), I think its good idea to connect it to that input (usually this is done by `aria-describedby` and `aria-invalid`. This gets a little complicated when you are using uncontrolled inputs + formRef + FormData (which are all vanilla Javascript rather than React) but its doable. It might be out of the scope of what is possible in our current time constraints however.

Reply:
We are maybe thinking of making a modal instead of the alerts.

I have been reading up about ARIA and we will def add this in!